tutorials/05-linear-regression/05_linear-regression.jmd

```diff
@@ -65,7 +64,7 @@ The next step is to get our data ready for testing. We'll split the `mtcars` dat
 select!(data, Not(:Model))
 
 # Split our dataset 70%/30% into training/test sets.
-trainset, testset = splitobs(shuffleobs(data), 0.7)
+trainset, testset = map(DataFrame, splitobs(data; at=0.7, shuffle=true))
```

Comment:
Great to know the `splitobs` has an argument `shuffle` - it looks quite similar to `sklearn.model_selection.train_test_split(shuffle=True)`.

Reply:
There are many other alternatives. I guess if you like scikit-learn (I only use it when I'm forced to for teaching 😉), then you might prefer MLJ.

Reply:
> There are many other alternatives. I guess if you like scikit-learn (I only use it when I'm forced to for teaching wink), then you might prefer MLJ.

Thanks a lot for mentioning MLJ - it's a very interesting interface. Yes I used sklearn/tidyverse/dplyr in the early days back to 17/18 for quick model deployment, so feel quite curious about similar syntax in Julia. Thanks for the illustrations, very well demonstrated.